_index.md

```diff
@@ -14,4 +14,5 @@ Community discussions take place on the
 [`SPECs` Discourse forum](https://discuss.scientific-python.org/c/specs/6).
 SPEC development takes place in the [SPEC repository](https://github.com/scientific-python/specs).
 
+If you want to **contribute a SPEC**, start by reading [SPEC Purpose and Process](/specs/purpose-and-process).
```

Comment:
Any use in adding a paragraph here that points to where to start if you want to **endorse** a SPEC?

Mostly just thinking about making that pathway clearly distinct from **contribution** and also a little more discoverable?

Reply:
See 7558a711b8d26633ce36534bddcb8b4d38c269ba